pkg/scheduling/volumeusage.go

```diff
@@ -132,7 +122,7 @@ func resolveDriver(ctx context.Context, kubeClient client.Client, volumeName str
 	}
 	if storageClassName != "" {
 		driverName, err := driverFromSC(ctx, kubeClient, storageClassName)
-		if err != nil {
+		if err != nil && !errors.IsNotFound(err) {
```

Comment:
We should think about how to solve the `NotFound` issue wholistically for all of the storage resolution logic that happens in `state.Cluster`. As it stands right now: if there is ever an error that is persistent and continues to happen (like a NotFound error) that is going to hang the cluster state while it's waiting for that node to get propagated.

Reply:
We resolved this offline, the current approach will be to ignore volume limits for a pod when we encounter `NotFound` errors. This ensures that we can update cluster state and progress.